data/sample_srcflux_plugins.py

```diff
@@ -366,3 +366,48 @@ def spectral_fit_sample_srcflux_obsid_plugin(infile, outroot, band, elo, ehi, sr
 
     return retval
     
+
+
+def aplimits_srcflux_obsid_plugin(infile, outroot, band, elo, ehi, src_num):
+    """
+    Sample plugin: aplimits
+    
+    This plugin runs the new `aplimits` script to compute
```

Comment:
```suggestion
    This plugin runs the `aplimits` script to compute
```

Very substantial comment, I know.